cmd/apiserver/app/server/run_server.go

```diff
@@ -31,15 +31,15 @@ import (
 )
 
 // RunServer runs an API server with configuration according to opts
-func RunServer(opts *ServiceCatalogServerOptions) error {
+func RunServer(opts *ServiceCatalogServerOptions, stopCh <-chan struct{}) error {
```

Comment:
why an arg vs in options?

Reply:
The options are established before the determination is made about which component to run. The channel is not available until the component is run, so we would have to add the channel to the options at that point. Moreover, the channel is not truly an option, and it does not seem appropriate to force it into the options struct.